browser/static/js/main_track.js

```diff
@@ -831,9 +831,17 @@ function action(action, info, frame = current_frame) {
       if (payload.imgs) {
         // load new value of seg_array
         // array of arrays, contains annotation data for frame
-        seg_array = payload.imgs.seg_arr;
-        seg_image.src = payload.imgs.segmented;
-        raw_image.src = payload.imgs.raw;
+        if (payload.imgs.hasOwnProperty('seg_arr')) {
+          seg_arr = payload.imgs.seg_arr;
```

Comment:
The js variable name is `seg_array` not `seg_arr`; this should either be changed here to `seg_array` or could be changed elsewhere for consistency (same thing in main_zstack.js)

Reply:
good catch!